package/rancher-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# first arg is `-flag` or `--some-flag`
+if [ "${1:0:1}" = '-' ]; then
```

Comment:
why do we have this check @janeczku? in which case is /usr/bin/rancher-entrypoint.sh called with flags?

Just wondering why we have this and the check below, can you please explain?